app/controllers/prompts_controller.rb

```diff
@@ -145,9 +145,10 @@ def create
 
   def update
     if @prompt.update_attributes(prompt_params)
-      flash[:notice] = 'Prompt was successfully updated.'
+      flash[:notice] = ts("Prompt was successfully updated.")
       redirect_to collection_signup_path(@collection, @challenge_signup)
     else
+      flash[:error] = ts("This change does not comply with sign-up rules, sorry. For more details, try using \"Edit Sign-up\".")
```

Comment:
Instead of adding this error message, I think you should be able to display the detailed errors on the prompt view, using this helper:

https://github.com/otwcode/otwarchive/blob/030fa4607d3c59fba2cacbe4e9c99b3c49fbc1cb/app/helpers/validation_helper.rb#L32-L42

For example, in the Edit Sign-up form:

https://github.com/otwcode/otwarchive/blob/030fa4607d3c59fba2cacbe4e9c99b3c49fbc1cb/app/views/challenge_signups/_signup_form.html.erb#L35

Reply:
Thanks for the pointer! I'm new to Rails, so I'm slowly finding my way around. I changed things accordingly and updated the description & testing instructions above.